test/test_holiday_jp.rb

```diff
@@ -30,4 +30,20 @@ def test_mountain_day_from_2016
       assert HolidayJp.holiday?(Date.new(year, 8, 11))
     end
   end
+
+  def test_custom_holiday
+    my_holiday_1 = Date.new(2015, 5, 13)
+    my_holiday_2 = Date.new(2015, 12, 7)
+
+    assert !HolidayJp.holiday?(my_holiday_1)
+    assert !HolidayJp.holiday?(my_holiday_2)
+
+    HolidayJp.add_custom_holiday_sources File.expand_path('../custom_holidays_1.yml', __FILE__), File.expand_path('../custom_holidays_2.yml', __FILE__)
```

Comment:
Line is too long. [151/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.